setup.py

```diff
@@ -106,7 +106,7 @@ def run(self):
               'scipy'
           ],
           extras_require={
-              'gui': ['ipywidgets >=8.0.0', 'voila']
+              'gui': ['ipywidgets <=7.7.1', 'ipympl<0.9', 'voila']
```

Comment:
Why are we enforcing an old version of `ipywidgets` and `ipympl`? Shouldn't we be trying to role with the updates / bug fixes of these dependencies?

Reply:
We found that the new release of `ipywidgets` is not compatible with `voila`. So until that's fixed we're kind of stuck